packages/universal-components/src/MenuGroup/MenuGroup.js

```diff
@@ -39,6 +41,10 @@ export default function MenuGroup({
 }
 
 const styles = StyleSheet.create({
+  title: {
```

Comment:
`MenuGroupTitle` is used only inside `MenuGroup` so I think we can also move this style directly inside `MenuGroupTitle`.

Reply:
Done